addons/website_hr_recruitment/controllers/__init__.py

```diff
@@ -1 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
this is actually not needed anylonger in python3, as utf-8 is already a default encoding. So you can remove it. 

You might still see in some files, which is fine, does not hurt, but at least we don't add it to new files

Reply:
Noted!